tracer/contrib/redigo/tracedredis_test.go

```diff
@@ -0,0 +1,161 @@
+package tracedredis
+
+import (
+	"context"
+	"github.com/DataDog/dd-trace-go/tracer"
+	"github.com/garyburd/redigo/redis"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+const (
+	debug = false
+)
+
+func TestClient(t *testing.T) {
+	assert := assert.New(t)
+	testTracer, testTransport := getTestTracer()
+	testTracer.DebugLoggingEnabled = debug
+
+	c, _ := TracedDial("my-service", testTracer, "tcp", "127.0.0.1:6379")
+	c.Do("SET", 1, "truck")
+
+	testTracer.FlushTraces()
+	traces := testTransport.Traces()
+	assert.Len(traces, 1)
+	spans := traces[0]
+
+	assert.Len(spans, 1)
+	span := spans[0]
+	assert.Equal(span.Name, "redis.command")
+	assert.Equal(span.Service, "my-service")
+	assert.Equal(span.Resource, "SET")
+	assert.Equal(span.GetMeta("out.host"), "127.0.0.1")
+	assert.Equal(span.GetMeta("out.port"), "6379")
+	assert.Equal(span.GetMeta("redis.raw_command"), "SET 1 truck")
+	assert.Equal(span.GetMeta("redis.args_length"), "2")
+}
+
+func TestError(t *testing.T) {
```

Comment:
You probably need another test that yields an error when connecting to the service, making sure if the underlying library can not connect to redis (bad credentials, bad host, whatever) it bubbles up to the caller.